pip_api/_parse_requirements.py

```diff
@@ -292,7 +292,9 @@ def _parse_editable(editable_req):
     original_url = url_no_extras
 
     if os.path.isdir(original_url):
-        if not os.path.exists(os.path.join(original_url, "setup.py")):
+        pyproject_path = os.path.join(original_url, "pyproject.toml")
+        setup_path = os.path.join(original_url, "setup.py")
+        if not any(os.path.exists(p) for p in (pyproject_path, setup_path)):
             raise PipError(
                 "Directory %r is not installable. File 'setup.py' not found."
```

Comment:
If we're changing the behavior here, we should update this error message as well.

(Is this strictly required for us to adopt `pyproject.toml` for this project?)

Reply:
Yep -- this path gets exercised against this project's own sdist, so this function ends up failing due to the sdist now containing a `pyproject.toml` instrad of `setup.py`. I'll update the error message!